config/helmchart/values.yaml.tpl

```diff
@@ -33,9 +33,9 @@ env:
 
 kube_rbac_proxy:
   image:
-    repository: quay.io/coreos/kube-rbac-proxy
+    repository: registry.redhat.io/openshift4/ose-kube-rbac-proxy
```

Comment:
the other operators have the google image here. with your approach if ones tries this operator in an OKD cluster they will fail.